miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/controller/EditPoolController.java

```diff
@@ -319,4 +318,83 @@ protected void writeConfiguration(ObjectMapper mapper, ObjectNode config) {
   public ModelAndView editPools(@RequestParam("ids") String poolIds, ModelMap model) throws IOException {
     return bulkEditBackend.edit(poolIds, model);
   }
+
+  private final BulkMergeTableBackend<PoolDto> bulkMergeBackend = new BulkMergeTableBackend<PoolDto>(
+      "pool", PoolDto.class, "Pool", "Pools") {
+
+    /** Given a bunch of strings, find the long substring that matches all of them that doesn't end in numbers or underscores. */
+    private String findCommonPrefix(String[] str) {
```

Comment:
Make this a static method in `LimsUtils` instead of duplicating from the `EditLibraryController`